### ISSUE TITLE
Support Service Discovery when configuring network

### ISSUE THREAD

**Issue by HUSSTECH** [aws/amazon-ecs-cli#485]
### Summary
It appears as if the current version of the CLI does not support service discovery yet. When bringing a service down then up again, it is not recreated with its service discovery network configuration in place.

### Description

- ecs-cli version: 1.4.2
- go version: 1.9.5
- platform: Ubuntu 16.04 and Debian Jessie

I had manually created an ecs service from the console, so that I could set up service discovery manually first to try it out. This was created correctly, and was working as expected. Then when I needed to update the service with a new task definition, I used `service down` then `service up` via the cli. When the service was back up again, the service discovery settings were not present on the service anymore.

### Config files
* ecs-params.yml
```
version: 1
task_definition:
  ecs_network_mode: awsvpc

run_params:
  network_configuration:
    awsvpc_configuration:
      subnets:
        - subnet-xxxxxx
      security_groups:
        - sg-xxxxxxx
```

### Observed Behavior
service is correctly brought down, then up again. But upon re-creation, if there was already ECS service discovery configured, it is no longer present. i.e. There is no route53 entry. As if it was removed. You have to use the console to recreate the service with service discovery.

**Comment by mhonig** [comment]
Hi @SoManyHs,

Do you have an ETA for service discovery support in the ecs-cli? Is there any workaround that we can use in the meantime if we want to use service discovery with our ecs-cli managed services?

Regards,
Moritz

**Comment by mhonig** [comment]
Hi @SoManyHs,
Thank you for the quick reply and I will keep an eye on your project board.
It was/is my understanding that service discovery can only be configured for an ECS Service upon creation of the same but not when it is updated. The AWS CLI [update-service](https://docs.aws.amazon.com/cli/latest/reference/ecs/update-service.html) does not support the service-registry parameter. That is why now I use the ECS CLI to create the task definition only and then create the service separately using the AWS CLI.
Once you support the service-registry parameter in your _ecs-cli compose service_ command I can combine these two steps in one.

**Comment by SoManyHs** [comment]
Hi @mhonig,

Thanks for clarifying. Yes, our plan is to incorporate service discovery into the `compose service` command to streamline the workflow for launching a service.

**Comment by SoManyHs** [comment]
Hi @FLemon, we are actively working on integrating service discovery and have plans to make it available to customers soon. In the meantime, feel free to follow the [service discovery branch](https://github.com/aws/amazon-ecs-cli/tree/servicediscovery) for updates. Thanks!

**Comment by PettitWesley** [comment]
@HUSSTECH @mhonig @FLemon Support for Service Discovery was just released in [1.9.0](https://github.com/aws/amazon-ecs-cli/releases/tag/v1.9.0)!
